### PR TITLE
SBFT'24 Competition - No mutation testing

### PR DESCRIPTION
**Pull request by phi-go**

### CODE REVIEW

**Comment by Alan32Liu** [comment]
Oh scratch my comment in the other PR.
Will run this now

> @Alan32Liu could you try another coverage experiment with this branch?

**Comment by Alan32Liu** [comment]
/gcbrun run_experiment.py -a --experiment-config /opt/fuzzbench/service/experiment-config.yaml --experiment-name sbft-standard-cov-01-17 --fuzzers libafl libfuzzer mystique pastis bandfuzz tunefuzz fox

**Comment by Alan32Liu** [comment]
ops, did you forget to make the dummy change to trigger exp? : )
Unfortunately I don't have the permission to add it to your branch.

**Comment by Alan32Liu** [comment]
/gcbrun run_experiment.py -a --experiment-config /opt/fuzzbench/service/experiment-config.yaml --experiment-name sbft-standard-cov-01-17 --fuzzers libafl libfuzzer mystique pastis bandfuzz tunefuzz fox

**Comment by phi-go** [comment]
@Alan32Liu, done, it was in the sbft branch that I removed ...

It also seems that we do not have a report for pastis, so we could skip that fuzzer to save some time in the eval.

/gcbrun run_experiment.py -a --experiment-config /opt/fuzzbench/service/experiment-config.yaml --experiment-name sbft-standard-cov-01-18 --fuzzers libafl libfuzzer mystique bandfuzz tunefuzz fox

**Comment by phi-go** [comment]
Ah you were faster, how long would you expect this experiment to take?

**Comment by Alan32Liu** [comment]
> Ah you were faster, how long would you expect this experiment to take?

1-2 days?
I will need to re-run the command because the last one was somehow interrupted.

**Comment by Alan32Liu** [comment]
/gcbrun run_experiment.py -a --experiment-config /opt/fuzzbench/service/experiment-config.yaml --experiment-name sbft-standard-cov-01-18 --fuzzers libafl libfuzzer mystique bandfuzz tunefuzz fox

**Comment by phi-go** [comment]
Oh, in that case let's include pastis. Sorry for the back and forth.

**Comment by phi-go** [comment]
/gcbrun run_experiment.py -a --experiment-config /opt/fuzzbench/service/experiment-config.yaml --experiment-name sbft-standard-cov-01-18 --fuzzers libafl libfuzzer mystique pastis bandfuzz tunefuzz fox

**Comment by Alan32Liu** [comment]
Experiment `sbft-standard-cov-01-18` data and results will be available later at:
[The experiment data](https://storage.googleapis.com/fuzzbench-data/index.html?prefix=sbft-standard-cov-01-18/).
[The experiment report](https://storage.googleapis.com/www.fuzzbench.com/reports/sbft-standard-cov-01-18/index.html).

**Comment by Alan32Liu** [comment]
/gcbrun run_experiment.py -a --experiment-config /opt/fuzzbench/service/experiment-config.yaml --experiment-name sbft-standard-cov-01-18-p --fuzzers libafl libfuzzer mystique pastis bandfuzz tunefuzz fox

**Comment by phi-go** [comment]
Thank you!

**Comment by Alan32Liu** [comment]
Experiment `sbft-standard-cov-01-18-p` data and results will be available later at:
[The experiment data](https://storage.googleapis.com/fuzzbench-data/index.html?prefix=sbft-standard-cov-01-18-p/).
[The experiment report](https://storage.googleapis.com/www.fuzzbench.com/reports/sbft-standard-cov-01-18-p/index.html).

**Comment by phi-go** [comment]
@Alan32Liu we had one final commit... Could you start the experiment again?
Also did something go wrong with the experiment, I don't see any build directories?

**Comment by Alan32Liu** [comment]
/gcbrun run_experiment.py -a --experiment-config /opt/fuzzbench/service/experiment-config.yaml --experiment-name sbft-standard-cov-01-19-p --fuzzers libafl libfuzzer mystique pastis bandfuzz tunefuzz fox

**Comment by phi-go** [comment]
@Alan32Liu can you post the link to the results of the experiment as well? 🙂

**Comment by Alan32Liu** [comment]
This is so strange: The experiment was not requested.
image

This also happened on the previous 2 exps.
Let me try again.

**Comment by Alan32Liu** [comment]
/gcbrun run_experiment.py -a --experiment-config /opt/fuzzbench/service/experiment-config.yaml --experiment-name sbft-standard-cov-01-20 --fuzzers libafl libfuzzer mystique pastis bandfuzz tunefuzz fox

**Comment by Alan32Liu** [comment]
Pretty sure it is working this time!

Experiment `sbft-standard-cov-01-20` data and results will be available later at:
[The experiment data](https://storage.googleapis.com/fuzzbench-data/index.html?prefix=sbft-standard-cov-01-20/).
[The experiment report](https://storage.googleapis.com/www.fuzzbench.com/reports/experimental/sbft-standard-cov-01-20/index.html).
OR
[The experiment report](https://storage.googleapis.com/www.fuzzbench.com/reports/bft-standard-cov-01-20/index.html).
